src/install.sh

```diff
@@ -514,12 +521,37 @@ function upload_custom_credentials() {
   fi
 }
 
+function create_k8s_port_forward() {
+  if [ "$SERVICE_TYPE" != "ClusterIP" ]; then
+    return
+  fi
+
+  nginx_pod=$(kubectl $KUBECTL_OPTIONS get pods -o=custom-columns=NAME:.metadata.name | grep nginx | tail -1)
+  cat <<EOL
+
+********************************************************************************
+
+ ClusterIP service type requires port forwarding using 'kubectl port-forward'
+ Please run the following command in another shell:
+
+
+ sudo kubectl $KUBECTL_OPTIONS port-forward $nginx_pod 80:80
```

Comment:
sudo?

Reply:
yah, ideally we use another port but that was going to be harder.